### PR TITLE
Fix server closable

### PR DESCRIPTION
**Pull request by jm-willach**
The method _listen()_ in the interface _AsyncServer_ should return a Closeable, but because of the coroutine or launch, the Return is not reached. See below:

	suspend fun listen(handler: suspend (AsyncClient) -> Unit): Closeable {
		val job = coroutineScope {
			launchImmediately {
				while (true) {
					handler(accept())
				}
			}
		}
		return Closeable { job.cancel() }
	}

To fix that, we added to the Interface :AsyncClosable. 
And the class _JvmAsyncServer_ contains the method _close()_.

### CODE REVIEW

File: korio/build.gradle [hunk 1]
```diff
@@ -7,7 +7,7 @@ dependencies {
 
 	commonMainApi "org.jetbrains.kotlinx:kotlinx-coroutines-core:$coroutinesVersion"
 
-	add("androidMainApi", "org.jetbrains.kotlinx:kotlinx-coroutines-android:$coroutinesVersion")
+//	add("androidMainApi", "org.jetbrains.kotlinx:kotlinx-coroutines-android:$coroutinesVersion")
```
Comment (soywiz) [inline korio/build.gradle]: Why did you comment this libe?

Comment (jm-willach) [reply korio/build.gradle]: Sorry its not necessary for the fix. That change was needed for me to sync gradle.

Comment (soywiz) [reply korio/build.gradle]: Can you add a commit adding it again?

**Comment by soywiz** [comment]
Thanks!

**Pull request merged**